php-checks/src/main/java/org/sonar/php/checks/TrailingWhitespaceCheck.java

```diff
@@ -55,19 +56,20 @@ public void setCharset(Charset charset) {
   }
 
   @Override
-  public void visitFile(AstNode astNode) {
+  public void visitCompilationUnit(CompilationUnitTree tree) {
     List<String> lines;
     try {
-      lines = Files.readLines(getContext().getFile(), charset);
+      lines = Files.readLines(context().file(), charset);
     } catch (IOException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)